src/series7/Problem1.java

```diff
@@ -0,0 +1,11 @@
+package series7;
+
+public class Problem1 {
+	public static void main(String[] args)
+	{
+		int a=5, b=3;
```

Comment:
Can you write code as:

`
int a = 5, b = 3;
`

Basically add space after the variable and after the equals sign as well